tests/OmniSharp.DotNetTest.Tests/RunTestFacts.cs

```diff
@@ -223,5 +223,27 @@ public async Task RunMSTestStandardOutputIsReturned()
             Assert.Single(response.Results);
             Assert.NotEmpty(response.Results[0].StandardOutput);
         }
+
+        [Fact]
+        public async Task RunMSTestWithRunSettings()
+        {
+            await RunDotNetTestAsync(
+                MSTestProject,
+                methodName: "Main.Test.MainTest.CheckRunSettings",
+                testFramework: "mstest",
+                shouldPass: true,
+                useRunSettings: true);
+        }
+
+        [Fact]
+        public async Task RunMSTestWithoutRunSettings()
+        {
+            await RunDotNetTestAsync(
```

Comment:
if this is supposed to fail, could you assert on some failure conditions to distinguish a "failure by design" from "failure because something got broken"?

Reply:
Added some additional assertions.